geode-benchmarks/src/test/java/org/apache/geode/benchmark/topology/ClientServerTopologyWithSniProxyTest.java

```diff
@@ -44,9 +46,11 @@ public void afterEach() {
 
   @ParameterizedTest
   @EnumSource(SniProxyImplementation.class)
+  @ClearSystemProperty(key = WITH_SNI_PROXY_PROPERTY)
   public void configWithNoSsl(final SniProxyImplementation sniProxyImplementation) {
-    TestConfig testConfig = new TestConfig();
-    ClientServerTopologyWithSNIProxy.configure(testConfig, sniProxyImplementation);
+    System.setProperty(WITH_SNI_PROXY_PROPERTY, sniProxyImplementation.name());
```

Comment:
We shouldn't be "tunneling" parameters this way. We should only be accessing system properties in some sort of main or launcher which then sends them either individually or as part of config structures down to the other software that needs the values.

Reply:
See previous comment... agree but out of scope.